try:
    import Run_fb_enc
    Run_fb_enc.main()
except Exception as e:
    print(f"An error occurred: {e}")
